tests/sals/automated_chatflows/test_vdc_chatflows.py

```diff
@@ -0,0 +1,35 @@
+import gevent
+from tests.base_tests import BaseTests
+import pytest
+from jumpscale.loader import j
+from solutions_automation.vdc import deployer
+from tests.sals.vdc.vdc_base import VDCBase
+from parameterized import parameterized_class
+
+
+@parameterized_class(("flavor"), [("silver",), ("gold",), ("platinum",), ("diamond",)])
+@pytest.mark.integration
+class VDCChatflows(VDCBase):
+    flavor = "silver"
+
+    def test_01_deploy_vdc(self):
+        """Test case for deploying a VDC.
+
+        **Test Scenario**
+
+        - Deploy a VDC.
+        - Check that VDC is reachable.
+        - Delete a VDC
+        """
+
+        self.info("Deploy a VDC")
+        vdc_name = self.random_name().lower()
+        password = self.random_string()
+        vdc = deployer.deploy_vdc(vdc_name, password, self.flavor.upper())
+
+        self.info("Check that VDC is reachable")
+        request = j.tools.http.get(f"http://{vdc.vdc.threebot.domain}", verify=False, timeout=60)
```

Comment:
to check if it could get cert
```suggestion
        request = j.tools.http.get(f"http://{vdc.vdc.threebot.domain}", timeout=60)
```